spec/features/saml/saml_spec.rb

```diff
@@ -199,6 +199,8 @@
         click_on 'Yes'
 
         complete_idv_profile_ok(user.reload)
+        click_submit_default
+        enter_correct_otp_code_for_user(user)
```

Comment:
I understand that this is because we had to modify the `fill_out_phone_form_ok` helper to use `(555) 555-5555` so that the proofer gem returns a successful confirmation. I don't think we should have to modify the spec to test this feature. By tying the successful path to a specific number, that means we can no longer test the 2 paths: 1) where the user chooses to keep their existing number, and 2) where they configure a new number for proofing.